lib/IAMClient.js

```diff
@@ -287,24 +294,36 @@ class VaultClient {
      * Update Quota of an account
      *
      * @param {string} accountName - account name
-     * @param {number} quota - maximum quota for the account
+     * @param {bigint | number} quota - maximum quota for the account
      * @param {VaultClient~requestCallback} callback - callback
      * @returns {undefined}
      */
     updateAccountQuota(accountName, quota, callback) {
+        const quotaIsValid = (typeof quota === 'bigint' || typeof quota === 'number') && quota > 0;
```

Comment:
can't we set `quota` to 0 disable account quota?

Reply:
No, we have a dedicated API for that now: DeleteAccountQuota. We already reject in Update if we get 0 (in the API).